app/views/organizations/home/index.html.erb

```diff
@@ -72,54 +72,64 @@
     <!--//? section header -->
     <div class="d-flex justify-content-start mb-2">
       <h2 class="section-heading text-uppercase text-xl">
-        How to apply for adoption
+        <%= t('.how_to_apply') %>
       </h2>
     </div>
 
     <!--//? cards section - info (not bootstrap card)  -->
     <div class="d-flex flex-wrap justify-content-around align-items-center px-4">
 
-      <div class="d-flex flex-column justify-content-center align-items-center">
-        <div class="mb-3">
-          <%= image_tag('paw-account.svg', width: '150', alt: "Create and account") %>
-        </div>
-        <div>
-          <p class="text-black">
-            1. Create Account
-          </p>
-        </div>
-      </div>
-      <div class="d-flex flex-column justify-content-center align-items-center">
-        <div class="mb-3">
-          <%= image_tag('paw-profile.svg', width: '150', alt: "Complete questionnaire") %>
-        </div>
-        <div>
-          <p class="text-black">
-            2. Complete Questionnaire
-          </p>
-        </div>
-      </div>
-      <div class="d-flex flex-column justify-content-center align-items-center">
-        <div class="mb-3">
-          <%= image_tag('paw-browse.svg', width: '150', alt: "Browse the pets") %>
+      <ol class="d-flex flex-row justify-content-around align-items-start w-100 gap-4">
+        <div class="d-flex flex-column justify-content-center align-items-center">
+          <div class="mb-3">
+            <%= image_tag('paw-account.svg', width: '150', alt: "Create an account") %>
+          </div>
+          <div>
+            <p class="text-black">
+              <li>
+                <%= t('.create_account') %>
+              </li>
+            </p>
```

Comment:
I don't believe this `p` tag is helping us. Can you just add the class to `li` and remove `p`? 

Also I didn't notice this before but can you double check that the text/number is center aligned to the icon above? It seems like the text is aligned but not including the number. Probably happened when switching to the ordered list. If this suggestion doesn't look right please say so.

Maybe there is a bootstrap class that can help, if not adding a style attribute such as `list-style-position: inside;` on the `li`

Reply:
I'll take another look, sure! Will agree that the p tag could probably just be done away with. But yes, I think this is being caused by the switch to the ordered list. I'll dig into it and hopefully have it done soon!